docs/en/Concept.md

```diff
@@ -0,0 +1,41 @@
+# Basic Concept                                
+##
+## 1 Message Model
+
+The RocketMQ message model is mainly composed of a Producer, a Broker and a Consumer.The Producer is responsible for producing messages, the Consumer is responsible for consuming messages, and the Broker is responsible for storing messages.     
```

Comment:
we don't use "a" upon on proper nouns generally.:
The RocketMQ message model is mainly composed of  Producer, Broker, and Consumer.